### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by yzhang175**
Guys, I have added a basic style for our website.
I have made changes for fonts, layouts, etc. and I also used W3.CSS framework for some. 
It is might not fancy, and not like what you imagined. lol
If you don't like it, feel free to edit it anytime.

Updates:
Since we can only use one framework. I have done couple things this time:
1. deleted the W3C framework & use plain html and css
2. changed a lot of divs, right now only couple divs in header 
3. deleted second css document, keep the css all in one file (style.css)

Updates on 3/16:
1. change divs
2. h1 problems
3. tried to fix the onclick event, but have not figure it out

### CODE REVIEW

**Review by hyang77 — DISMISSED** [review]
Looks good! Thanks for making our site beautiful :)

File: public/china/index.html [hunk 1]
```diff
@@ -4,41 +4,64 @@
...
+
+  <!-- Country Information Categories-->
   <main id="content">
-    <h1 id="country">China</h1>
-    <p id="fullname"></p>
 
-    <section id="language">
-      <h2>Language</h2>
+    <!-- Name can be hide later -->
+    <section class="">
```
Comment (wxie11) [inline public/china/index.html:46]: what is the class? If you are not using it, please delete it.

File: public/china/index.html [hunk 2]
```diff
@@ -4,41 +4,64 @@
...
-        <li><a href="../china/">China</a></li>
-        <li><a href="../japan/">Japan</a></li>
-        <li><a href="../malaysia/">Malaysia</a></li>
-        <li><a href="../singapore/">Singapore</a></li>
-        <li><a href="../south_korea/">South Korea</a></li>
-        <li><a href="../thailand/">Thailand</a></li>
-      </ul>
+    <!-- Sidebar -->
+    <nav class="sidebar barmenu topheader" id="mySidebar">
+      <a onclick="close_menu()" class="button">Close Menu</a>
```
Comment (wxie11) [inline public/china/index.html:13]: "written leveraging event handlers and wholly avoiding deprecated JavaScript attributes in your HTML, such as onload" -- please make sure onclick is not deprecated.

File: public/china/index.html [hunk 3]
```diff
@@ -4,41 +4,64 @@
...
+      <a href="#" class="button">Feedback</a>
+      <a href="#" class="button">About Us</a>
     </nav>
+
+    <!-- Top -->
+    <div class="fixedtop">
```
Comment (wxie11) [inline public/china/index.html]: Avoid div inside div

File: public/china/index.html [hunk 4]
```diff
@@ -4,41 +4,64 @@
...
+      <div class="topheader">
+        <div class="hambutton" onclick="open_menu()">☰</div>
+        <div class="logo"><a href="../index.html">Travelbriefing</a></div>
+      </div>
+
+      <div class="countrynavi" id="navigation">
```
Comment (wxie11) [inline public/china/index.html]: Why do we need a div here? Can we just add that class and id to the <nav> and style from there?

File: public/china/index.html [hunk 5]
```diff
@@ -47,28 +70,29 @@ <h2>Telephone</h2>
       </ul>
     </section>
 
-    <section id="water">
-      <h2>Drinking water</h2>
+    <section class="categories" id="water">
+      <h2 class="">Drinking water</h2>
       <p id="water_safety"></p>
-    </section>
+    </div>
```
Comment (wxie11) [inline public/china/index.html:77]: where does this </div> come from?

**Comment by wxie11** [comment]
I added a couple of comments, please address @yzhang175

**Review by wxie11 — APPROVED** [review]
nice work

**Review by jhuang60 — APPROVED** [review]
lgtm

**Pull request merged**